packages/app-builder-lib/src/codeSign/windowsCodeSign.ts

```diff
@@ -74,6 +115,39 @@ export async function sign(options: WindowsSignOptions, packager: WinPackager):
   return true
 }
 
+export async function signUsingAzureTrustedSigning(options: WindowsSignOptions, packager: WinPackager): Promise<boolean> {
+  const vm = await packager.vm.value
+
+  const ps = await getPSCmd(vm)
+  await vm.exec(ps, ["Install-Module", "-Name", "TrustedSigning", "-RequiredVersion", "0.4.1", "-Force", "-Repository", "PSGallery"])
+
+  const config: WindowsAzureSigningConfiguration = options.options.azureOptions!
+
+  const configFilter = config.FilesFolderFilter?.split(",") ?? []
+  const signExts = new Set(["exe", ...configFilter])
+  if (options.options.signDlls === true) {
+    signExts.add("dll")
+  }
+  if (options.options.signExts) {
+    options.options.signExts.forEach(v => signExts.add(v))
+  }
+
+  const params = {
+    ...config,
+    FilesFolder: options.path,
```

Comment:
`options.path` is a file, not a folder, right? We can pass it using `-Files` instead of `-FilesFolder` and `-FilesFolderFilter`.

Reply:
Great callout. Will update that.